### ISSUE TITLE
When my input_dir is a txt file, an error will be reported

### ISSUE THREAD

**Issue by xings19** [visual-layer/fastdup#135]
When my input_dir is a txt file, an error will be reported.
My code:
```python
import fastdup
fd = fastdup.create(input_dir='filename.txt',work_dir='/my/work/dir/')
result = fd.run(sync_s3_to_local=False,overwrite=True,delete_img=True)
```
Output error:
```text
Traceback (most recent call last):
  File "/mnt/lustre/username/.conda/envs/unpdata/lib/python3.8/site-packages/fastdup/sentry.py", line 114, in inner_function
    ret = func(*args, **kwargs)
  File "/mnt/lustre/username/.conda/envs/unpdata/lib/python3.8/site-packages/fastdup/fastdup_controller.py", line 307, in run
    self._init_run(input_dir, annotations, subset, embeddings, data_type, overwrite, fastdup_kwargs)
  File "/mnt/lustre/username/.conda/envs/unpdata/lib/python3.8/site-packages/fastdup/fastdup_controller.py", line 109, in _init_run
    self._verify_fastdup_run_args(input_dir, self._work_dir, df_annot, subset, self._dtype, embeddings)
  File "/mnt/lustre/username/.conda/envs/unpdata/lib/python3.8/site-packages/fastdup/fastdup_controller.py", line 953, in _verify_fastdup_run_args
    verify_input_dirs_or_s3_paths_exist(input_dir)
  File "/mnt/lustre/username/.conda/envs/unpdata/lib/python3.8/site-packages/fastdup/fastdup_controller.py", line 971, in verify_input_dirs_or_s3_paths_exist
    verify_single_path(input_dir_or_list)
  File "/mnt/lustre/username/.conda/envs/unpdata/lib/python3.8/site-packages/fastdup/fastdup_controller.py", line 965, in verify_single_path
    assert os.path.isdir(str(path)), f'input_dir {str(path)} is not a directory'
AssertionError: input_dir filename.txt is not a directory
```
Each line in the file is the path of a picture, and the picture is placed on aws, like this:
```text
s3://INTERN1.0/google/wiki/1/4074/1003/3641/282da628ffdffce0ebbe82403a677aa4eaeb3179.jpeg
s3://INTERN1.0/google/wiki/1/6657/523/5221/c2d9065995c2b36034fe75c5995e4d3c0fe1c31c.jpeg
s3://INTERN1.0/google/wiki/3/311/195/6838/afef0302c0cf9bb5b9614ee99c8e665805af0161.jpeg
s3://INTERN1.0/google/wiki/1/2028/2854/954/85b3c6817fe53e4d7cd6776e1a5a6fa73ef4b056.jpeg
s3://INTERN1.0/google/wiki/0/5424/3470/923/7f1bacb31ca0e4dca772fa2c235f02d4cf0138dc.jpeg
……
```

**Comment by dbickson** [comment]
hi @xings19 thanks for reporting. Submitting a list of s3 path is not supported yet in v1, we will add support in the coming days. In the meantime, as a workaround, you can work with 0.2X api:
```
import fastdup
fastdup.run(s3_input_list, work_dir=my_work_dir)

**Comment by dbickson** [comment]
This should be fixed in 0.910 please reopen if you encounter anything
